conanfile.py

```diff
@@ -0,0 +1,52 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class QtProtobufConan(ConanFile):
+    name = "qtprotobuf"
+    version = "0.4.0"
+    license = "MIT"
+    url = "https://github.com/semlanik/qtprotobuf"
+    description = ("gRPC and Protobuf generator and bindings for Qt framework")
+    topics = ("conan", "qt", "protobuf")
+    settings = "os", "compiler", "build_type", "arch"
+    homepage = "https://github.com/semlanik/qtprotobuf"
+    options = {"shared": [True, False]}
+    default_options = {
+        "shared": False,
+        "qt:qtdeclarative": True,
+    }
+    generators = "cmake"
+    requires = [
+        "protobuf/3.9.1",
+        "protoc_installer/3.9.1@bincrafters/stable",
+        "qt/5.14.2@bincrafters/stable",
+        "grpc/1.25.0@inexorgame/stable",
+        "openssl/1.1.1d",
```

Comment:
Looks like Qt already has dependency on openssl. We don't need to add it explicitly, because QtProtobuf doesn't have direct dependency on it

Reply:
It is used here to resolve dependency conflict between gRPC and Qt. They both depend on certain versions of openssl, so this entry sort of overrides the version.

Reply:
Since we don't need grpc anymore, let's remove openssl too.